docs/src/content/docs/reference/configuration.md

```diff
@@ -157,8 +160,11 @@ type SidebarItem = {
   translations?: Record<string, string>;
 } & (
   | { link: string }
-  | { items: SidebarItem[] }
-  | { autogenerate: { directory: string } }
+  | { items: SidebarItem[]; collapsed?: boolean }
+  | {
+      autogenerate: { directory: string; collapsed?: boolean };
```

Comment:
Definitely feeling pleased with myself for making `autogenerate` an object 😁 

I had a suspicion some day we might want stuff in addition to the directory!

Reply:
Great call indeed.